style.css

```diff
@@ -0,0 +1,2496 @@
+*{ 
+    margin: 0; 
+    padding: 0; 
+    box-sizing: border-box;
+    font-size: 16px;
+    font-family: Helvetica, Arial, sans-serif
+}
+
+body, main { 
+    width: 100%;
+    min-height: 100vh;
+    display: flex;
+    align-items: center; /* Centered children horizontally */
+    justify-content: space-around;
+    background-color: silver;
+}
+
+h1 {
+    display: flex;
+    justify-content: center;
+    font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+    font-size: 2rem;
+    margin-bottom: 2%;
+    color: #333;
+}
+
+.button{
+    padding: 10px 20px;
+    color: #333;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    margin: 5% 5% 5% 5%;
+    border: none;
+    border-radius: 4px;
+}
+
+#puzzle1{
+    background-color: #90EE90;
+}
+
+#puzzle2{
+    background-color: #FFA07A;
+}
+
+#puzzle3{
+    background-color: #FF6347;
+}
+
+#leave {
+    background-color: #4682B4;
+}
+
+#buttons {
+    display: flex;
+    justify-content: center;
+    list-style-type: none;
+    gap: 2px;
+}
+
+#start {
+    display: flex;
+    justify-content: center;
+    list-style-type: none;
+    padding: 10px 20px;
+    color: #333;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    margin: 5% 5% 5% 5%;
+    border: none;
+    border-radius: 4px;
+    background-color: #D8BFD8;
+}
+
+#reset {
+    display: none;
+    list-style-type: none;
+    padding: 10px 20px;
+    color: #333;
+    text-align: center;
+    text-decoration: none;
+    margin: 5% 5% 5% 5%;
+    border: none;
+    border-radius: 4px;
+    background-color: #D8BFD8;
+}
+
+.button:hover, .button:focus {
+    transform: translateY(-2px);
+    box-shadow: 0 4px 8px rgba(0,0,0,0.2); 
+}
+
+button:hover, button:focus {
+    transform: translateY(-2px);
+    box-shadow: 0 4px 8px rgba(0,0,0,0.2);
+} 
+
+@media (max-width: 767px) and (orientation: portrait) { /*Orientation property: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/orientation*/
+    
+#containers{ /*All puzzles*/
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 90vw;
+    height: 80vh;
+    border-radius: 20px;
+    gap: 2vh; 
+}
+
+#piece-container{ /*All puzzles*/
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    width: 70%; /*For all media queries, I toogled width and height each time until i was visually satisfied*/
+    height: calc(50% - 1vh); /*For all media queries, I toogled width and height each time until i was visually satisfied*/
+    border: 1px solid black;
+    position: relative;
+}
+
+#puzzle-container{ /*All puzzles*/
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    width: 70%; /*For all media queries, I toogled width and height each time until i was visually satisfied*/
+    height: calc(50% - 1vh); /*For all media queries, I toogled width and height each time until i was visually satisfied*/
```

Comment:
There are two choices here that may be causing issues: 1) the choice to use percentages in width and height units, and 2) the choice to define both width and height, rather than just one. Together these seem to be causing skewing when the viewport is taller or wider than the width and height at which it was coded.

![image](https://github.com/Birkbeck2/web-project-adeledsg/assets/47217308/c05d1ee1-3dd7-4e19-adc4-9af55502ff89)

 I suspect it is also making it more difficult to get the pieces to align without gaps.

Did you experiment with using rem or em units (with only x or y specified, not both)? It may be you'd have the same issues with that approach, but if not, it's worth a try.